playground/ssr-deps/package.json

```diff
@@ -26,6 +26,7 @@
     "@vitejs/test-non-optimized-with-nested-external": "workspace:*",
     "@vitejs/test-optimized-with-nested-external": "file:./optimized-with-nested-external",
     "@vitejs/test-optimized-cjs-with-nested-external": "file:./optimized-with-nested-external",
+    "@vitejs/test-nested-exclude": "file:./nested-exclude",
```

Comment:
For the test case, I copied the same fixture used in `playground/optimize-deps`

https://github.com/vitejs/vite/blob/c2cf80243b9f28075ecd7d7a2ebcef28c069af1d/playground/optimize-deps/package.json#L40